### ISSUE TITLE
Accuracy of MXNet training statistics

### ISSUE THREAD

**Issue by GaiYu0** [dmlc/mxnet#4694]
Hi, I am using MXNet evaluation metrics to acquire training statistics. I notice that the accuracy produced by mxnet.metric.Accuracy always contains 6 digits behind decimal point. I am using 5000 samples as validation data. Therefore, in validation accuracy, there should be at most 4 meaningful digits behind decimal point (1 / 5000 = 0.0002) and the last 2 digits, if present, should be 0. However, in that case, the last 2 digits are usually not 0, e.g. 0.887891, 0.890039, 0.888281, etc. I am using MXNet in a research project. Therefore, I might be slightly particular about details. Could anyone explain why the last 2 digits are not 0?

**Comment by piiswrong** [comment]
print out metric.sum_metric and metric.num_inst to see if they are integers

**Comment by GaiYu0** [comment]
Thanks for help! The issue occurs because metric.sum_metric is not equal to the number of validation samples if validation batch size cannot divide the number of samples.
